addons/event/models/event_slot_ticket.py

```diff
@@ -0,0 +1,98 @@
+from odoo import api, fields, models
+from odoo.tools.sql import drop_view_if_exists, SQL
+
+
+class EventSlotTicket(models.Model):
+    """ Used to keep track of the registrations per slot and per ticket for multi-slots events.
+    I.e. if the seats_max for a specific ticket is 5, keeping track of the ticket registrations for each slot
+    to ensure that it doesn't exceed 5. If the number of registrations reaches 5 for a specific slot,
+    the ticket will be considered sold out for that slot.
+    """
+    _name = "event.slot.ticket"
+    _description = "Event Slot Ticket"
+    _auto = False
+
+    event_id = fields.Many2one("event.event", string="Event", related="slot_id.event_id")
```

Comment:
Would probably store it as we are going to search / read_group on it frequently I bet